src/test/java/de/zalando/zmon/dataservice/proxies/entities/EntitiesControllerTest.java

```diff
@@ -74,7 +74,7 @@ public void getEntities() throws Exception {
 
     @Test
     public void getEntitiesRest() throws Exception {
-        this.mockMvc.perform(MockMvcRequestBuilders.get("/rest/api/v1/entities?query=htg").header("Authorization", "Bearer 1"))
```

Comment:
this one fixes the remaining usage: https://github.com/zalando-zmon/zmon-scheduler/pull/34